buildSrc/build.gradle.kts

```diff
@@ -19,43 +19,29 @@ buildscript {
 }
 
 plugins {
-    `java-gradle-plugin`
     `kotlin-dsl`
 }
 
-sourceSets {
-    main {
-        java.setSrcDirs(listOf("src"))
-    }
-    test {
-        java.setSrcDirs(listOf("tst"))
-    }
-}
+// Note: We can't use our standard source layout due to https://github.com/gradle/gradle/issues/14310
 
 dependencies {
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jacksonVersion")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
     implementation("org.eclipse.jgit:org.eclipse.jgit:5.0.2.201807311906-r")
-    implementation("com.atlassian.commonmark:commonmark:0.15.2")
+    implementation("org.commonmark:commonmark:0.17.1")
+
     implementation("software.amazon.awssdk:codegen:$awsSdkVersion")
+
     implementation("org.jetbrains.intellij.plugins:gradle-intellij-plugin:$ideaPluginVersion")
 
+    implementation("org.jlleitschuh.gradle:ktlint-gradle:10.0.0")
```

Comment:
do we want to move these versions to gradle.properties?

Reply:
We should, yes. At least till we get on 7.0 with version catalog files

Reply:
Cool, didn't know this was an upcoming feature (https://melix.github.io/blog/2021/03/version-catalogs.html), will be nice code cleanup when we can use it

Reply:
Yep, heres the release news: https://docs.gradle.org/7.0-rc-1/release-notes.html#centralized-versions